src/styles/bloks/cards.scss

```diff
@@ -0,0 +1,124 @@
+.cards {
+  --cards-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--cards-count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    & {
+      --cards-count: 2;
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      --cards-count: 3;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      --cards-count: 4;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  color: $color-nav;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  @include hover(transform, scale(1.2));
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: #616070;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    color: #616070;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-price {
+      color: $color-nav;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    color: $color-second;
+    text-transform: uppercase;
+    background-color: $main-color;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    display: flex;
+    border: 1px solid $main-color;
+
+    transition: all 0.3s;
+
+    &:hover {
+      color: $main-color;
+      background-color: $color-second;
+    }
+  }
```

Comment:
The button styles are very well implemented. However, the transition property is set to 'all'. This means that if you add another property that can be transitioned, it will also have the transition effect. It's usually better to specify which properties you want to transition. In this case, you probably only want to transition the background-color and color properties.